smdebug/core/tfevent/summary.py

```diff
@@ -92,7 +92,7 @@ def _make_histogram(values, bins, max_bins):
         limits = new_limits
 
     # Find the first and the last bin defining the support of the histogram:
-    cum_counts = np.cumsum(np.greater(counts, 0, dtype=np.int32))
+    cum_counts = np.cumsum(np.greater(counts, 0))
```

Comment:
Can we keep an int dtype here, or is the output of np.cumsum always an int? Want to make sure we have a consistent type.

Reply:
I've tested that `np.cumsum(np.greater(counts, 0))` outputs int64 regardless if inputs are of type bool or int32.